src/types/ERC725JSONSchema.ts

```diff
@@ -18,20 +18,8 @@ export type ERC725JSONSchemaValueContent =
   | 'Markdown'
   | string; // for tuples
 
-export type ERC725JSONSchemaValueType =
-  | 'string'
-  | 'address'
-  | 'uint256'
-  | 'bytes32'
-  | 'bytes'
-  | 'bytes4'
-  | 'string[]'
-  | 'address[]'
-  | 'uint256[]'
-  | 'bytes32[]'
-  | 'bytes4[]'
-  | 'bytes[]'
-  | string; // for tuples;
+// https://github.com/lukso-network/LIPs/blob/main/LSPs/LSP-2-ERC725YJSONSchema.md#valueType
+export type ERC725JSONSchemaValueType = string;
```

Comment:
maybe there is something we could do to separate tuples from other types by having `string: never` or something in ERC725JSONSchemaValueType and a separate type for tuples which is just a string? Then we can still have the security of the type as it was before while allowing tuples to be any string. 

Not too sure how this would look in TS just thinking out loud